src/main/java/UserModificationUI.java

```diff
@@ -0,0 +1,83 @@
+/**
+ * Provides the UI elements
+ */
+import javax.swing.*;
+import java.awt.GridLayout;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class UserModificationUI implements ChangeController{
+    private JLabel label;
+    UserDatabase db = UserDatabase(accounts);
```

Comment:
Hi Parmis, just wondering where is accounts defined in UserModificationUI? In the UserDatabase class, I see that it's a file but in the UserModificatinUI class I think accounts doesn't exist as a variable? And also I think there may be a missing new keyword? Maybe I'm just misunderstanding something, but the rest of it makes sense though, thank you for your work ~

Reply:
Hi! Yea Madhav mentioned that we can assign the value for accounts (a file) in a main method so it works a global varaible? I'm not too sure how it will workout but I'll make modifications again as necessary if that doesn't work!

Reply:
Ohh okay yeah that makes sense if it were global, sounds good thank youu!